code/game/objects/structures/electricchair.dm

```diff
@@ -63,7 +63,7 @@
 		return
 	if(!A.powered(EQUIP))
 		return
-	if(locked_atoms.len)
+	if(ISREALLIST(locked_atoms))
```

Comment:
This works as intended - shocks the human if he's buckled, otherwise just runs the
``` 
spark_system.start() //just something to let them know it works``` in the else